sites/hashdev/theme-override.d.ts

```diff
@@ -137,4 +137,4 @@ declare module "@mui/material/Paper" {
 }
 
 // https://github.com/mui-org/material-ui/issues/28244
-export default "";
+export {};
```

Comment:
Fixes `no-default-exports` but keeps the idea. https://github.com/mui/material-ui/issues/28244#issuecomment-1181448039 suggests so too.